src/js/components/Tag/Tag.js

```diff
@@ -50,8 +54,8 @@ const Tag = forwardRef(({ name, value, onRemove, onClick, ...rest }, ref) => {
           plain
           hoverIndicator
           focusIndicator
-          icon={<FormClose />}
-          round={theme.tag.round}
+          icon={<FormClose {...theme.tag.size?.[size]?.icon} />}
```

Comment:
Shouldn't this be `theme.tag.size?.[size]?.icon?.size`?

Reply:
We could do size={theme.tag.size?.[size]?.icon?.size} but the above lets you specify any icon property. Whichever you prefer...

Reply:
I like that this way allows you to specify any icon property just for our open-source mentality to not be to opinionated